src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -13,11 +13,19 @@ import com.jetbrains.python.psi.PyFormattedStringElement
 import com.jetbrains.python.psi.PyStringLiteralExpression
 import com.jetbrains.python.psi.types.TypeEvalContext
 
+/**
+ * If [searchRelativelyToCurrentFolder] is true - the file sought relative to the current folder
+ * else relative to the project root
+ * If [makePathRelativelyCurrentFolder] is true - the path built relative to the current folder
+ * else relative to the project root
+ */
```

Comment:
Please use KDoc notation for documentation, see https://kotlinlang.org/docs/kotlin-doc.html#generate-the-documentation

E.g. ide doesn't show documentation (F1 pressed on `searchRelativelyToCurrentFolder`) for current implementation

![image](https://user-images.githubusercontent.com/72933/125284430-1b384d80-e322-11eb-86e6-eb0b79168f80.png)

but will show if use KDoc notation, e.g. `@param`
![image](https://user-images.githubusercontent.com/72933/125284560-402cc080-e322-11eb-8c8b-295d309ebd59.png)

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/377/commits/77f71e32a664ea6f36b570865a81c466c60d0c76